routes/route_team.jsx

```diff
@@ -313,7 +313,7 @@ export default {
                     getComponents: (location, callback) => {
                         Promise.all([
                             System.import('components/team_sidebar'),
-                            System.import('components/sidebar.jsx'),
+                            System.import('components/sidebar'),
```

Comment:
Going to conflict with #501

Reply:
This is rebased, so the conflict is already fixed.